associacao-nacional-de-pesquisa-e-ensino-em-transportes.csl

```diff
@@ -412,7 +412,7 @@
       </else>
     </choose>
   </macro>
-  <citation et-al-min="6" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year" givenname-disambiguation-rule="primary-name">
+  <citation et-al-min="4" et-al-use-first="1" et-al-subsequent-min="4" et-al-subsequent-use-first="1" disambiguate-add-names="true" disambiguate-add-givenname="true" givenname-disambiguation-rule="primary-name" disambiguate-add-year-suffix="false" year-suffix-delimiter="; ">
```

Comment:
are you sure you want to remove the `collapse="year"` property? That would give you (Smith 1991, Smith 1992) instead of (Smith 1991, 1992). 
Also, you can remove `et-al-subsequent-min="4" et-al-subsequent-use-first="1" ` which is redundant now.